egs/dihard_2018/v2/conf/queue.conf

```diff
@@ -0,0 +1,10 @@
+# Default configuration
+command qsub -v PATH -cwd -S /bin/bash -j y -l arch=*64*
+option mem=* -l mem_free=$0,ram_free=$0
+option mem=0          # Do not add anything to qsub_opts
+option num_threads=* -pe smp $0
+option num_threads=1  # Do not add anything to qsub_opts
+option max_jobs_run=* -tc $0
+default gpu=0
+option gpu=0 -l 'hostname=b1[234568]*|c*,gpu=0' -q g.q
+option gpu=* -l 'hostname=b1[234568]*|c*,gpu=$0' -q g.q
```

Comment:
This is a site specific file and should be deleted.

Reply:
I know this file is explained in the documentation and in queue.pl, but it would have helped me to have had an example file under conf. Could this be made generic and part of the recipe?